src/components/JourneyComparison/JourneyComparison.tsx

```diff
@@ -13,12 +13,18 @@ import FareComparison from "./FareComparison";
 import styles from "./JourneyComparison.module.scss";
 
 const getTotalJourneyDuration = (journey: JourneyInfo) => {
+    if ("error" in journey) {
+        return 0;
```

Comment:
Returning `null` instead of `0` here might more readily signal that there was a problem.